lapce-app/src/terminal/raw.rs

```diff
@@ -54,7 +54,10 @@ impl RawTerminal {
         proxy: ProxyRpcHandler,
         term_notification_tx: Sender<TermNotification>,
     ) -> Self {
-        let config = alacritty_terminal::term::Config::default();
+        let config = alacritty_terminal::term::Config {
+            semantic_escape_chars: ",│`|\"' ()[]{}<>\t".to_string(),
```

Comment:
why not leave it to default?

Reply:
The default configuration includes the character ":", so when clicking, the line number and character number after ":" cannot be obtained. For example, with "src/main.rs:11:9", the terminal can only get "src/main.rs" and cannot get the subsequent "11:9".